content/docs/command-reference/add.md

```diff
@@ -6,16 +6,26 @@ file.
 ## Synopsis
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
-               [--glob] [--file <filename>] [-o <path>]
-               [--to-remote] [-r <name>] [-j <number>] [-f]
+usage: dvc add [-h] [-q | -v] [-f] [--no-commit] [--external]
+               [--glob] [-o <path>]
+               [--to-remote] [-r <name>] [--remote-jobs <number>]
                [--desc <text>] [--meta key=value] [--label <str>]
                [--type <str>]
                targets [targets ...]
 positional arguments:
   targets               Files or directories to add
 ```
 
+<details>
+
+### Options deprecated in 3.0
```

Comment:
```suggestion
### Options removed in 3.0
```

deprecate = discouraged to use but still supported.
These were removed in 3.0 without a replacement.

Reply:
I am not sure if we need to list these, as they don't exist anywhere else in the docs (like what it did in 2.0, etc).

And we don't support multiple versions in docs. :)

Reply:
I see we are already violating this in other PRs and it seems there's a lot of pushback, so I'll drop for now. I guess we could always come back and add these notices if we see confusion or people complain.